draft-thomassen-dnsop-generalized-dns-notify.md

```diff
@@ -470,11 +531,11 @@ Per {{!RFC8552}}, IANA is requested to create the "Generalized DNS
 Notifications" registry with the following columns and initial
 entries:
 
-| NOTIFY type | Location             | relative to | Reference       |
-| ----------- | -------------------- | ----------- | --------------- |
-| CDS         | _cds-notifications   | parent      | [this document] |
-| CSYNC       | _csync-notifications | parent      | [this document] |
-| DNSKEY      | TODO                 | TODO        | [this document] |
+| NOTIFY type | Scheme | Location | Reference       |
+| ----------- | ------ | -------- | --------------- |
+| CDS         |      1 | parent.  | [this document] |
+| CSYNC       |      1 | parent.  | [this document] |
+| DNSKEY      |      1 | zone.    | [this document] |
 
 [TODO: (1) Detail out use cases? (2) Add entries via RFC or web form?]
```

Comment:
Should be expert review

Reply:
Hmm? I don't understand your comment here?

Reply:
I commented on my own old TODO in line 540, which raises the question by what mechanism entries can later be added to the NOTIFY IANA registry, such as via an RFC, or via a simple web form.

It then occurred to me that we should say that it can be done based via Expert Review, which is one of the "accredited" mechanisms. 

If you agree, we can borrow language [here](https://www.ietf.org/archive/id/draft-ietf-dnsop-dns-catalog-zones-09.html#name-iana-considerations-2).